core/src/main/java/org/apache/calcite/sql/dialect/MssqlSqlDialect.java

```diff
@@ -189,6 +189,19 @@ public MssqlSqlDialect(Context context) {
     }
   }
 
+  @Override public void unparseBoolLiteral(SqlWriter writer,
+      SqlLiteral literal, int leftPrec, int rightPrec) {
+    Boolean value = (Boolean) literal.getValue();
+    if (value == null) {
+      return;
+    }
+    final SqlWriter.Frame frame = writer.startList("(", ")");
```

Comment:
Is the code here the same as that in Oracle Dialect? If so, it can be extracted using a static method called RelToSqlConverterUtil.

Reply:
Done.